CONTRIBUTING.md

```diff
@@ -0,0 +1,50 @@
+# Contributing
+
+Please start by discussing the change via [issue](https://github.com/LDSSA/curriculum-development/issues).
+
+# Code of Conduct
+
+LDSSA's [Code of Conduct](https://github.com/LDSSA/wiki/wiki/Code-of-Conduct).
+
+# Teaching Philosophy
+
+LDSSA's [Teaching Philosophy](https://github.com/LDSSA/wiki/wiki/Teaching-Philosophy).
+
+# Pull Request Process
+
+We use the [GitHub flow](https://guides.github.com/introduction/flow/).
+
+1. Create a new branch, following the naming convention `ISSUE_NUMBER/ISSUE_TITLE`
+2. Update the file(s) with the proposed changes
+3. Commit the proposed changes
+4. Open a Pull Request to discuss the changes.
+
+# Approval Process
+
+1. Anyone on `#curriculum` can discuss, review, or upvote/downvote Pull Requests
+2. Reviews by Teaching, Quality Assurance, and Student Success are optional but encouraged
```

Comment:
Gabo should provide some input here.

Reply:
Quality Assurance is optional at the moment, do you think it should be mandatory?

Reply:
Ah well for curriculum and not necessarily code it should be okay. If something slips through the cracks then we can add QA to the process.